build/building/env-vars.md

```diff
@@ -0,0 +1,129 @@
+---
+title: Environment variables for Docker Build
+description: Use environment variables to configure builds
+keywords: docker build, buildx, buildkit, env, environment variables, config
+---
+
+You can set the following environment variables to enable, disable, or change
+the behavior of features related to building:
+
+| Variable                                                          | Type              |
+| ----------------------------------------------------------------- | ----------------- |
+| [BUILDX_BUILDER](#buildx_builder)                                 | String            |
+| [BUILDX_CONFIG](#buildx_config)                                   | String            |
+| [BUILDX_EXPERIMENTAL](#buildx_experimental)                       | Boolean           |
+| [BUILDX_GIT_INFO](#buildx_git_info)                               | Boolean           |
+| [BUILDX_GIT_LABELS](#buildx_git_labels)                           | String \| Boolean |
+| [BUILDX_NO_DEFAULT_ATTESTATIONS](#buildx_no_default_attestations) | Boolean           |
+| [BUILDX_NO_DEFAULT_LOAD](#buildx_no_default_load)                 | Boolean           |
```

Comment:
Almost forgot we have [other env vars](https://grep.app/search?q=BUILDKIT_&case=true&filter[repo][0]=docker/buildx&filter[lang][0]=Go) we should add here:

* [`BUILDKIT_HOST`](https://github.com/docker/buildx/blob/f2ac30f431e96d7be3e7ba7497f3f1802995db0e/commands/create.go#L109): introduced by https://github.com/docker/buildx/pull/1078 (cc @jedevc)
* `BUILDKIT_PROGRESS`: https://docs.docker.com/engine/reference/commandline/buildx_build/#progress
* `BUILDKIT_COLORS`: https://docs.docker.com/build/buildkit/color-output-controls/

Reply:
Should we split the `BUILDX` and `BUILDKIT` vars into separate tables? IMO, it looks a bit odd with them bundled all together, we should explain to users why some have different prefixes.

Reply:
I do agree on the inconsistency here but I think it's fine and not worth extra explanation. For example `BUILDKIT_HOST` is only used in buildx and should have been smth like `BUILDX_BUILDKIT_HOST` but `BUILDKIT_PROGRESS` is used on both buildx and buildkit.

Reply:
given the inconsistencies that @crazy-max points out here, I think we could run with a single table for now. I think it would be nice if we could avoid implementation details like BUILDX in the variable prefix in the future. Especially with buildx being the default this could be a source of confusion. Why not make `DOCKER_BUILD_xxx` the standard prefix for build variables for example?

Reply:
> Why not make `DOCKER_BUILD_xxx` the standard prefix for build variables for example?

Agree, `DOCKER_` prefix would make more sense: https://grep.app/search?q=DOCKER_&case=true&filter[repo][0]=docker/cli&filter[lang][0]=Go